### PR TITLE
fixed bugs

### PR DESCRIPTION
**Pull request by aEmirturkoglu**
pr for bugfixes

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const express = require("express");
 const app = express();
 const { createHash } = require('crypto');
 const fs = require('fs');
-const PORT = process.env.PORT;
+const PORT = process.env.PORT /*maybe put || 3000 or something like that*/;
 const db = require("./db.js");
```
Comment (anuragdaksh7) [inline app.js:7]: yes

File: database.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ async function addUser(Email, username, password){
     let collection = db.collection("web-app");
     let response = await collection.find({email:Email}).toArray();
     if (response.length > 0) return "USER ALREADY EXISTS!";
-    else await collection.insertOne({ email: Email, user: username, pass: password  });
+    else await collection.insertOne({ email: Email, user: username, pass: password  }); //maybe use bcrypt here like pass: hashedpassword
```
Comment (anuragdaksh7) [inline database.js:23]: I already hashed it (this is hashed password)
image
- ss from the DB

**Review by anuragdaksh7 — APPROVED** [review]

File: database.js (hunk 2 above)
Comment (aEmirturkoglu) [reply database.js:23]: wow i didnt know that it could be hashed fram a db

Comment (anuragdaksh7) [reply database.js:23]:
```javascript

        const { createHash } = require('crypto');

        function hash(string) {
             return createHash('sha256').update(string).digest('hex');
        }
        console.log(hash(password))

``` 

I used this to hash in app.js

**Pull request merged**